### PR TITLE
@mzikherman => [Artist Page] Add pagination A/B test

### PR DESCRIPTION
**Pull request by damassi**
This PR adds a new split test, `ARTIST_PAGE_PAGINATION`, which when set to `experimental` will disable infinite scroll and add a pagination UI control: 

scroll

- http://localhost:5000/artist/keith-haring?split_test[artist_page_pagination]=experiment

#### TODO: 
- Enable A/B test; right now we're behind a `ENABLE_EXPERIMENTAL_ARTIST_PAGINATION` flag

### CODE REVIEW

**Comment by mzikherman** [comment]
This looks great, perfect to me! 👌 

In terms of deploying + analytics- sometimes we do a deploy with the AB test commented out (and a feature flag or query param in its place). This lets us still deploy safely and not block the pipeline, while also QA'ing the new behavior, and any additional analytics that were needed. Then, at the end when we're ready to launch the AB test, we have a PR that removes the custom query param/lab feature/flag in favor of the proper AB test as you've set up here.

This seems pretty innocuous, but I can see @katarinabatina and @briansw wanting to test the pagination out without launching the AB test (or blocking Force deploys in the meantime). What do you think about commenting out the AB test stuff in favor of a lab feature/flag (I can help w/ lab feature if you want)?

Alternatively, we can ask in #product-discover, maybe they're ok just launching it.

File: src/desktop/components/artwork_filter_2/view.coffee [hunk 1]
```diff
@@ -68,6 +78,7 @@ module.exports = class ArtworkFilterView extends Backbone.View
     @$htmlBody.animate { scrollTop: visibleTop - 1 }, 500
 
   infiniteScroll: =>
+    return if @infiniteScrollEnabled is false
```
Comment (mzikherman) [inline src/desktop/components/artwork_filter_2/view.coffee]: #minor can be `return unless @infiniteScrollEnabled`

**Comment by katarinabatina** [comment]
Let’s put t behind a flag we can’t quite launch the test yet due to our search test.

**Comment by mzikherman** [comment]
Nice!

**Pull request merged**